src/display.rs

```diff
@@ -96,6 +97,12 @@ impl Fail {
                 true,
                 String::from("(OK:DISABLED) Distribution -> NetBSD system detected, so the distribution is automatically hidden."),
             ),
+            #[cfg(target_os = "macos")]
+            distro: FailedComponent::new(
+                true,
+                String::from("(OK:DISABLED) Distribution -> macOS system detected, so the \
```

Comment:
@grtcdr Distribution messages for macOS is already here.

Reply:
So when does "oh no" appear?